src/compute/protocol.rs

```diff
@@ -175,7 +189,10 @@ pub struct Server {
     pub description: Option<String>,
     // TODO(dtantsur): flavor in newer versions
     pub flavor: Ref,
-    #[serde(deserialize_with = "empty_as_default", rename = "config_drive")]
+    #[serde(
+        deserialize_with = "bool_from_config_drive_string",
```

Comment:
I wonder if you could use `empty_as_default` here. `Default` for `bool` is `false`, which seems exactly what you need.

Also is it actually a string `"True"` or a proper boolean value `true`?

Reply:
It surprised me that it wasn't a boolean - https://docs.openstack.org/api-ref/compute/?expanded=create-server-detail,show-server-details-detail#show-server-details says `config_drive | body | string | Indicates whether or not a config drive was used for this server. The value is True or an empty string. An empty string stands for False.`

Reply:
The compute API is full of legacy (like using empty strings instead of null), sigh. Okay, I just wanted to double-check.